admin-events/src/utils.js

```diff
@@ -0,0 +1,26 @@
+const { ethers } = require('forta-agent');
+
+function getAbi(abiName) {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const { abi } = require(`../abi/${abiName}`);
+  return abi;
+}
+
+function getEventSignatures(contract, events) {
+  // given an ethers contract object and a list of event names, return a list of full event
+  // signatures
```

Comment:
I'm guessing we use this in other places, but it could be shored up.   The comment is incorrect, because we aren't using an ethers.js Contract Object, we're using our own custom rolled contract Object.

It seems like two general purpose functions would be useful, although they would not both be used in this particular bot:
`getEventSignaturesFromContract(contract, events)`
`getEventSignaturesFromInterface(iface, events)`

The first function would be a small wrapper that destructures the `Interface` Object from an `ethers.js` Contract Object,  calls the second function, and returns its returned Array.